src/SumoCoders/FrameworkCoreBundle/Resources/views/Form/fields.html.twig

```diff
@@ -114,6 +114,14 @@
                           <span class=\"pull-left date-separator\">/</span>",
           '{{ day }}': "<div class=\"col-md-2\">#{day_widget}</div>",
         })|raw }}
+      {% elseif date_pattern == '{{ month }}{{ day }}{{ year }}' %}
```

Comment:
is there no way to do this in the php, this is starting to look like way too much logic in the templates

Reply:
fixed, we always use the same format 745bd51